pkg/output/stdout.go

```diff
@@ -155,7 +160,12 @@ func (s *Stdoutput) resultQuiet(resp ffuf.Response) {
 }
 
 func (s *Stdoutput) resultNormal(resp ffuf.Response) {
-	res_str := fmt.Sprintf("%s%-23s [Status: %s, Size: %d, Words: %d]", TERMINAL_CLEAR_LINE, resp.Request.Input, s.colorizeStatus(resp.StatusCode), resp.ContentLength, resp.ContentWords)
+	res_str := fmt.Sprintf("%s %3d   %-8d   %-5d   %s", TERMINAL_CLEAR_LINE, s.colorizeStatus(resp.StatusCode), resp.ContentLength, resp.ContentWords, resp.Request.Input)
+	fmt.Println(res_str)
+}
+
+func (s *Stdoutput) resultUrl(resp ffuf.Response) {
+	res_str := fmt.Sprintf("%s %3d   %-8d   %-5d   %s", TERMINAL_CLEAR_LINE, s.colorizeStatus(resp.StatusCode), resp.ContentLength, resp.ContentWords, resp.Request.Url)
```

Comment:
`colorizeStatus` returns `string` because it adds colors using terminal escape sequences.
```suggestion
	res_str := fmt.Sprintf("%s %3s   %-8d   %-5d   %s", TERMINAL_CLEAR_LINE, s.colorizeStatus(resp.StatusCode), resp.ContentLength, resp.ContentWords, resp.Request.Url)
```